Assets/General/Scripts/MenuFunctionality/AudioSettings.cs

```diff
@@ -0,0 +1,169 @@
+/**********************************************************************************************************************
+// File Name :         AudioSettings.cs
+// Author :            Charlie Polonus
+// Creation Date :     11/17/24
+// 
+// Brief Description : Handles the audio settings menu, applying it to the VSAs
+**********************************************************************************************************************/
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using FMOD.Studio;
+using System.IO;
+
+public class AudioSettings : MonoBehaviour
+{
+    private VCA _vca;
+
+    [SerializeField] private string _audioSettingsFilePath;
+
+    [SerializeField] private Slider _masterSlider;
```

Comment:
might be best to put tooltips here just incase someone miss interprets something.